src/components/dialogs/event-dialog.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import React from 'react';
+
+import PropTypes from 'prop-types';
+
+import EventsTable from '../events-table';
+import { FormattedMessage } from 'react-intl';
+
+import {
+    Button,
+    Dialog,
+    DialogContent,
+    DialogTitle,
+    LinearProgress,
+    Typography,
+} from '@mui/material';
+import { Close } from '@mui/icons-material';
+
+const style = {
+    dialogTitleStyle: {
+        display: 'flex',
+        alignItems: 'center',
+        justifyContent: 'space-between',
+    },
+    dialogContentStyle: {
+        maxHeight: '70vh',
+    },
+};
+
+function EventDialog({ open, onClose, isLoadingEvent, taskData }) {
+    return (
+        <Dialog open={open} onClose={onClose} maxWidth="lg" fullWidth>
+            <DialogTitle sx={style.dialogTitleStyle}>
+                <Typography variant="h6" component="h2">
```

Comment:
Not sure whether you need this Typography element or not. It seems like the DialogTitle is already a H2 looking like H6, but maybe I'm wrong

Reply:
No the default value of variant props is body1 and body1 is smaller than h6.
Here is the extract from material ui documentation

![Capture d’écran de 2024-02-01 16-19-49](https://github.com/farao-community/gridcapa-app/assets/116628858/53448c65-5da4-47cb-95b9-800c9da7023c)